src/Workspaces/Core/Portable/Classification/AbstractClassificationService.cs

```diff
@@ -77,6 +78,7 @@ public async Task AddSyntacticClassificationsAsync(Document document, TextSpan t
             }
 
             var syntaxTree = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+            Contract.ThrowIfNull(syntaxTree);
```

Comment:
??

Reply:
We pass in null to F#/Typescript implementations?